suite-native/module-settings/src/components/ColorSchemePickerItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ColorValue, TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { colorVariants, ThemeColorVariant } from '@trezor/theme';
+import { Box, Text } from '@suite-native/atoms';
+
+type ColorSchemePickerItemProps = {
+    isSelected?: boolean;
+    onPress: () => void;
+    colorScheme: ThemeColorVariant;
+};
+
+const pickerItemWrapperStyle = prepareNativeStyle<{ isSelected: boolean }>(
+    (utils, { isSelected }) => ({
+        backgroundColor: utils.colors.gray100,
+        borderRadius: utils.borders.radii.medium,
+        height: 114,
+        flex: 1,
+        paddingTop: 33,
+        borderWidth: utils.borders.widths.medium,
+        borderColor: isSelected ? utils.colors.green : utils.colors.gray100,
+    }),
+);
+
+type PickerItemDotStyleProps = {
+    backgroundColor: ColorValue;
+    isFirstItem: boolean;
+};
+const pickerItemDotStyle = prepareNativeStyle<PickerItemDotStyleProps>(
+    (utils, { backgroundColor, isFirstItem }) => ({
+        width: 26,
+        height: 26,
+        backgroundColor,
+        borderRadius: utils.borders.radii.round,
+        extend: {
+            condition: !isFirstItem,
+            style: {
+                marginLeft: -utils.spacings.small,
```

Comment:
nit:  I would maybe prefer `* -1` which seems little bit more visible than just `-`, but I will leave this up to you

Reply:
Yeah I think it makes it more obvious. If we want to make it super clear, it would be nice to have a function for this. If it was something like `convertNumberToNegative(utils.spacings.small)` it would be clear to the developer also why this operation looks a little bit weird.
[d8023c3](https://github.com/trezor/trezor-suite/pull/5693/commits/d8023c3e5ae32f8c647643a1fb505ca9bbecc3ef)